### ISSUE TITLE
admin sessions

### ISSUE THREAD

**Issue by xLink** [Cysha/pxcms-admin#1]
come up with a way of seperating the admin sessions from the frontend, this will make it so you have to reauth or auth with an admin account instead

**Comment by NoelDavies** [comment]
@xLink reply?

**Comment by xLink** [comment]
even if we don't add a separate admin session, there should atleast be a "sudo" mode, ie if your auto logged in, you need to put your passy in to make account/system changes

**Comment by NoelDavies** [comment]
Yeah I get you, would be cool to have 2fa on that, to make cookie stealing impossible.

**Comment by xLink** [comment]
Since the `L5.3` upgrade, we can use the guards to solve this issue fairly easily...would need to repurpose the login stuff slightly to work with the new guard too...but should be fine :>

**Comment by xLink** [comment]
oh and the `auth.admin` middleware
